mp-ts.sh

```diff
@@ -48,4 +48,4 @@ clean)
    echo "Usage: $0 {dist-zip|dist-run}"
 esac
 
-exit 0 
```

Comment:
I'm no sure what was fixed here. Shouldn't it be part of different PR?

Reply:
Nothing was fixed, but my IDE wont let me "fix" it and return back those trailing spaces :)

Reply:
I'll try to squash commits, make git reset and commit without changes to this file...